src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,60 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+import java.util.Objects;
+
+/**
+ * Represents an adding or deleting of a tag for a specific person during a session
+ */
+public class Tagging {
+
+    public static final String SYMBOL_ADD = "+";
+    public static final String SYMBOL_REMOVE = "-";
```

Comment:
If you are already using enums for `Change`, you can also use a `Map<Enum, String>` to convert from enums to your symbol. This way, the `toString` method will not need to care if enums are added or removed (e.g. new enum for renaming tags).